### ISSUE TITLE
Deploying to new VPC with AWS Terraform installer no longer works

### ISSUE THREAD

**Issue by cricci82** [coreos/tectonic-installer#145]
**BUG REPORT**

**Tectonic version**:
1.5.5-2 Terraform Installer

**Platform** (bare-metal, AWS, local VMs):
AWS

**What happened**:
Running the 'plan' or 'apply' commands when deploying to a new VPC results in the following error:
```
module.vpc.data.aws_availability_zones.azs: Refreshing state...
module.masters.data.aws_vpc.cluster_vpc: Refreshing state...
module.vpc.data.aws_vpc.cluster_vpc: Refreshing state...
Error refreshing state: 2 error(s) occurred:

* data.aws_vpc.cluster_vpc: multiple VPCs matched; use additional constraints to reduce matches to a single VPC
* data.aws_vpc.cluster_vpc: multiple VPCs matched; use additional constraints to reduce matches to a single VPC
make: *** [plan] Error 1
```

**What you expected to happen**:
Install to complete with no errors

**How to reproduce it** (as minimally and precisely as possible):
Install following AWS Terraform instructions.  Be sure to create a new VPC

**Anything else do we need to know**:
This worked previously.

**Comment by sir-rob** [comment]
I ran into this as well.

**Comment by rji** [comment]
See #151.

**Comment by alexsomesan** [comment]
Merged #151 
Closing this.
